src/static/themes/auto/auto.css

```diff
@@ -1,2 +1,23 @@
-@import '../default/default.css';
-@import '../dark/dark.css' (prefers-color-scheme: dark);
+@import "../default/default.css";
+@import "../dark/dark.css" (prefers-color-scheme: dark);
+
+@media (forced-colors: active) {
```

Comment:
This appears to work for me, but now you mention it, I'm not sure how it works. I had thought auto was the default theme.

If I run `yarn run run; yarn run build` then the new instance that starts up, it follows my OS dark/light setting by default. That is Firefox UI does, tridactyl UI does and tridactyl new tab content all show up dark with my usual settings, but if I switch the os to light and restart it  they all follow that as well.

Is that picking up a local setting I have? I can't see anything relevant in my tridactylrc, and the Firefox that starts up doesn't seem to have any of my other settings or plugins in it's core settings.

Reply:
I strongly suspect a rogue tridactylrc somewhere, what does `:viewconfig --user` say? It should be empty except for lastupdatecheck and some other internal stuff

Reply:
Possibly a side effect of this change: https://github.com/tridactyl/tridactyl/pull/4115/files, since the tridactly UI css will always pull in auto, which will be the same as default (and auto) if your OS is asking for a light UI, but equal to auto and dark if you ask for a dark UI in your OS.

Reply:
I thought that change would get overridden by the user's chosen theme soon after loading? I thought it was trying to fix the #2510 style issues

Reply:
viewconfig --user only has:

{"update":{"lastchecktime":1656614748993},"configversion":"2.0"}

The use of auto does get overriden by the users theme choice to some degree (e.g. the dark gray background becomes a darker black for Quake theme users), but if Firefox is configured to opt into dark content offered by sites (which seems to be the default on the Mac  *if* your OS is set up to use dark mode) then the tridactyl content will switch to dark mode (which I think is a good thing) and some of that color will also bleed through to the tridactyl UI too, even if you explicitly select a 'light' theme like quakelight, or 'default'.

Which is possibly a bug, if the idea is to allow people to opt to have dark web content but still choose a light tridactyl theme at the same time, though whether anyone actually wants that I'm not sure. I was assuming that it would be expected that if the user opts for dark content then they'd want a dark tridactyl UI to match.